components/textfield/metadata/textfield.yml

```
@@ -1,6 +1,6 @@
 name: Text field
 status: Verified
-description: A single-line text field (aka textfield)
+description: A single-line text field using the `<input>` element`.
```

```suggestion
description: A single-line text field using the `<input>` element.
```

I noticed on the site there's a stray back-tick.

---

Ope good catch!